src/inventory_ui.cpp

```diff
@@ -558,22 +817,24 @@ void inventory_selector::draw( WINDOW *w ) const
         : _( "Item selection; [TAB] switches mode, arrows select." );
     const nc_color msg_color = ( navigation == navigation_mode::CATEGORY ) ? h_white : c_ltgray;
 
-    if( center_align ) {
+    if( are_columns_centered() ) {
         center_print( w, getmaxy( w ) - 1, msg_color, msg_str.c_str() );
     } else {
         trim_and_print( w, getmaxy( w ) - 1, 1, getmaxx( w ), msg_color, msg_str.c_str() );
     }
 }
 
-inventory_selector::inventory_selector( player &u, const std::string &title, const item_location_filter &filter )
+inventory_selector::inventory_selector( const player &u, const inventory_selector_preset &preset )
     : u(u)
+    , preset( preset )
```

Comment:
No, single-line initializer list is way less readable. We've discussed it [before](https://github.com/CleverRaven/Cataclysm-DDA/pull/17098#r66518264).